README.md

```diff
@@ -42,6 +42,12 @@ Command line:
      
      # Renders and compile templates to STDOUT 
      $ bin/cop --render-template examples/views/Dockerfile.hbs examples/setting.json examples/setting.yml
+     
+     # Accept configuration input from STDIN
+     $ curl https://raw.githubusercontent.com/jasmith590/COP/develop/examples/setting.yml | bin/cop --shell --stdin-type=yaml
+     
+     # Accept template input from STDIN
+     $ curl https://raw.githubusercontent.com/jasmith590/COP/develop/examples/views/Dockerfile.hbs | bin/cop --stdin-type=hbs examples/setting.json
```

Comment:
@jasmith590 do you think type should be the key? e.g. type=shell  type=handlebars  type=yaml   ? else it's kind of ambiguous, and type=conf|vars|env  make no sense to me to hint 'shell'.